VMAccess/vmaccess.py

```diff
@@ -67,6 +67,10 @@ def install():
 def enable():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error)
     hutil.do_parse_context('Enable')
+
+    if hutil.get_extension_version() == "1.4.5.0":
+        _forcefully_reset_chap()
```

Comment:
@boumenot Why limit to 1.4.5.0? Also, does this change increment the hot fix version?